index.html

```diff
@@ -9,7 +9,7 @@ <h2 class="text-center">What is Student Robotics?</h2>
         <img srcset="{{ '/images/advertising/team-in-arena@1x.jpg 1x' | prepend: site.baseurl }},
                       {{ '/images/advertising/team-in-arena@2x.jpg 2x' | prepend: site.baseurl }},
                       {{ '/images/advertising/team-in-arena@3x.jpg 3x' | prepend: site.baseurl }}"
-              src="{{ '/images/advertising/team-in-arena@3x.jpg' | prepend: site.baseurl }}"
+          src="{{ '/images/advertising/team-in-arena@3x.jpg' | prepend: site.baseurl }}"
```

Comment:
Whitespace change seems unrelated? Also seems to have changed the indentation away from lining up with the other attribute.

I suggest if we're going to go for a fixed-indent style rather than the current align-indent style that we wrap all the attributes:
``` html
<foo
  attr="blah"
  attr2="blah"
/>
```